test/modules/auxiliary/test/sqli_test.rb

```diff
@@ -28,20 +27,23 @@ def initialize(info = {})
         Opt::RHOST('127.0.0.1'),
         OptInt.new('RPORT', [true, 'The target port', 1337]),
         OptString.new('TARGETURI', [true, 'The target URI', '/']),
-        OptInt.new('SQLI_TYPE', [true, '0)Regular. 1) BooleanBlind. 2)TimeBlind', 0]),
+        OptInt.new('SQLI_TYPE', [true, '0)Regular. 1) BooleanBlind. 2)TimeBlind.', 0]),
         OptBool.new('SAFE', [false, 'Use safe mode', false]),
         OptString.new('ENCODER', [false, 'an encoder to use (hex for example)', '']),
         OptBool.new('HEX_ENCODE_STRINGS', [false, 'Replace strings in the query with hex numbers?', false]),
-        OptInt.new('TRUNCATION_LENGTH', [true, 'Test SQLi with truncated output (0 or negative to disable)', 0])
+        OptInt.new('TRUNCATION_LENGTH', [true, 'Test SQLi with truncated output (0 or negative to disable)', 0]),
+        OptInt.new('DBMS', [ true, '0)MySQL/MariaDB. 1) PostgreSQL. 2) Sqlite. 3) MSSQL.', 0])
```

Comment:
I wonder if using `OptEnum` here would help improve readability/tab completion/validation/etc; Likewise with `SQLI_TYPE`

Reply:
Updated. For `TRUNCATION_LENGTH`, It's expected to be a positive integer, so I kept `OptInt` for it.